packages/block-library/src/social-link/edit.native.js

```diff
@@ -193,4 +195,15 @@ const SocialLinkEdit = ( {
 	);
 };
 
-export default SocialLinkEdit;
+export default compose( [
+	withSelect( ( select, { clientId } ) => {
+		const { getBlock } = select( 'core/block-editor' );
+
+		const block = getBlock( clientId );
+		const name = block.name.substring( 17 );
```

Comment:
While testing removing some social links I got a red screen related to `block.name` can we use `?.` or check if the block exists before accessing to `name` ?